algorithms/scaling/combine_intensities.py

```diff
@@ -144,7 +144,7 @@ def _determine_Imids(self, raw_intensities):
             avg = flex.mean(raw_intensities)
             Imid = flex.max(raw_intensities) / 10.0
             Imid_list = [0, 1, avg, Imid]
-            while Imid > avg:
+            while (Imid > avg) and (Imid > 10):
```

Comment:
What is the significance of the value `10` here? A comment explaining the reasoning might be useful.

Reply:
I'll add a comment to the text and update the newsfragment once the CI job has finished.
Basically, what we're doing here is testing a value midpoint value across the span of the intensities (midpoint meaning for crossing from summation to profile intensities). So if the max intensity is 50K, for the midpoints we try mean, 5K, 500, 50 and 5. So the value of 10 means the last value to test is in the 1>10 range, which seems a sensible place to stop (but we only get down to this point if the mean is below 10).

Reply:
I wonder if `1` would be a bit less magical-looking and still as useful?

Reply:
Also maybe `while I > max(avg, 10):` would be clearer - setting an explicit limit? It's kinda got quite drastically different output behaviour for `avg=0+eps` and `avg=0-eps`